api/src/models/Customer.ts

```diff
@@ -0,0 +1,35 @@
+import {
+    CreationOptional,
+    InferAttributes,
+    InferCreationAttributes,
+    Model,
+    Sequelize,
+    DataTypes,
+  } from "sequelize";
+  import path from "path";
+  
+  interface CustomerModel
+    extends Model<
+      InferAttributes<CustomerModel>,
+      InferCreationAttributes<CustomerModel>
+    > {
+    // Some fields are optional when calling UserModel.create() or UserModel.build()
+    id: CreationOptional<string>;
+  }
+  
+  // Exportamos una funcion que define el modelo
+  // Luego le injectamos la conexion a sequelize.
+  module.exports = (sequelize: Sequelize) => {
+    // defino el modelo
+    sequelize.define<CustomerModel>(
+      path.basename(__filename, path.extname(__filename)).toLowerCase(),
+      {
+        id: {
+          type: DataTypes.UUID,
+          defaultValue: DataTypes.UUIDV4,
+          primaryKey: true,
+        },
+      }, 
+      { timestamps: false },
```

Comment:
Todas las tablas tienen que ser tablas paranoicas (por el requerimeinto del borrador logico.)
```suggestion
      { 
        timestamps: false,
        paranoid: true,
      },
```